beacon_node/store/src/hot_cold_store.rs

```diff
@@ -2075,16 +2062,15 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
             })??;
 
         if already_pruned && !force {
-            info!(self.log, "Execution payloads are pruned");
+            info!("Execution payloads are pruned");
             return Ok(());
         }
 
         // Iterate block roots backwards to the Bellatrix fork or the anchor slot, whichever comes
         // first.
         warn!(
-            self.log,
-            "Pruning finalized payloads";
-            "info" => "you may notice degraded I/O performance while this runs"
+            info = "you may notice degraded I/O performance while this runs",
+            "Pruning finalized payloads"
```

Comment:
same as above w.r.t. to a longer structured field than the log message. 

```suggestion
            info = "Pruning finalized payloads",
            "you may notice degraded I/O performance while this runs"       
```